ldap/admin/src/logconv.py

```diff
@@ -608,246 +621,617 @@ def _init_regexes(self):
+
+        # Tracking key
+        conn_scope_key = log_entry.get("key", "")
+        try:
+            op_scope_key = self.build_op_scope_key(conn_scope_key, op_id)
+        except (ValueError, TypeError) as e:
+            self.logger.error(f"Invalid key format for conn_scope_key={conn_scope_key}, op_id={op_id} - {e}")
+            op_scope_key = None
+
+        if isinstance(timestamp_dt, datetime.datetime):
+            timestamp_str = timestamp_dt.isoformat()
+        else:
+            timestamp_str = "Invalid timestamp"
+
+        # Should we ignore this operation
+        if conn_scope_key in self.connection.exclude_ip:
+            return None
+        
+        # Global counters
+        self.result.counters['result'] +=  1
+        self.result.counters['timestamp'] += 1
+
+        # Operation time fields
+        if etime and isinstance(etime, str):
+            try:
+                etime_f = float(etime)
+                heapq.heappush(self.result.etime_duration, etime_f)
+                if len(self.result.etime_duration) > self.size_limit:
+                    heapq.heappop(self.result.etime_duration)
+                self.result.total_etime += etime_f
+            except ValueError:
+                self.logger.debug(f"Invalid etime format: {etime}")
+
+        if wtime and isinstance(wtime, str):
+            try:
+                wtime_f = float(wtime)
+                heapq.heappush(self.result.wtime_duration, wtime_f)
+                if len(self.result.wtime_duration) > self.size_limit:
+                    heapq.heappop(self.result.wtime_duration)
+                self.result.total_wtime += wtime_f
+            except ValueError:
+                self.logger.debug(f"Invalid wtime format: {wtime}")
+
+        if optime and isinstance(optime, str):
+            try:
+                optime_f = float(optime)
+                heapq.heappush(self.result.optime_duration, optime_f)
+                if len(self.result.optime_duration) > self.size_limit:
+                    heapq.heappop(self.result.optime_duration)
+                self.result.total_optime += optime_f
+            except ValueError:
+                self.logger.debug(f"Invalid optime format: {optime}")
+
+        # Stat reporting to csv file
+        try:
+            self.result.etime_stat = round(self.result.etime_stat + float(etime), 8)
+        except (ValueError, TypeError):
+            self.logger.debug(f"Invalid etime for: {etime}")
 
-        if restart_conn_op_key in self.search.scope_rst_con_op_map:
-            del self.search.scope_rst_con_op_map[restart_conn_op_key]
+        # Track error if present
+        if err is not None:
+            self.result.error_freq[err] += 1
 
-        if restart_conn_op_key in self.search.filter_rst_con_op_map:
-            del self.search.filter_rst_con_op_map[restart_conn_op_key]
+        # Internal operation
+        if internal_op:
+            self.operation.counters['internal'] +=1
```

Comment:
minor style issue: should have space before 1

Reply:
Done